ForgeUnity/Assets/BeardedManStudios/Scripts/NetworkManager.cs

```diff
@@ -711,5 +728,76 @@ public GameObject GetGameObjectByNetworkId(uint id)
 
             return ((NetworkBehavior)foundNetworkObject.AttachedBehavior).gameObject;
         }
-    }
+
+		public virtual void SceneUnloaded(Scene scene)
```

Comment:
Please add the missing xmldoc documentation for this method.

Reply:
Done